src/DetailsView/details-view-body.tsx

```diff
@@ -72,7 +73,13 @@ export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
 
     private renderCommandBar(): JSX.Element {
         const { deps, switcherNavConfiguration } = this.props;
-        return <switcherNavConfiguration.CommandBar actionMessageCreator={deps.detailsViewActionMessageCreator} {...this.props} />;
+
+        const detailsViewCommandBarProps: DetailsViewCommandBarProps = {
```

Comment:
I know this was there before but since you are touching these components, it might make sense to move the action message creator into deps inside the DetailsViewCommandBarProps.

Reply:
I agree with the idea here, but actionMessageCreator shows up in many, many, props, and this feels like a change thought we should consider holistically. I'll look into it as a separate PR, but it doesn't feel like it belongs in this one